iron-dropdown.html

```diff
@@ -242,7 +242,10 @@
         get _horizontalAlignTargetValue() {
           var target;
 
-          if (this.horizontalAlign === 'right') {
+          // In RTL, the direction flips, so what is "right" in LTR becomes "left".
+          var isRTL = window.getComputedStyle(this).direction == 'rtl';
```

Comment:
Can we refactor `window.getComputedStyle(this).direction == 'rtl'` into an accessor that is used in both places? It would be nice to be able to easily swap out the test for `isRTL` in one place if we come up with a better way some day.